From 7fa07211320e4cb00d537844720edbf2b4fd047d Mon Sep 17 00:00:00 2001
From: Rex Dieter <rdieter@math.unl.edu>
Date: Thu, 27 Aug 2015 11:13:40 -0500
Subject: [PATCH] Add Requires.private: glib-2.0

since commit 6f616d4c4b04d5478f60d0311317bc84c5459ea to nm-version.h
a dependency to glib was added, but this wasn't reflected in pkg-config
deps... until now.
---
 NetworkManager.pc.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/NetworkManager.pc.in b/NetworkManager.pc.in
index 877c344..a0557ee 100644
--- a/NetworkManager.pc.in
+++ b/NetworkManager.pc.in
@@ -9,5 +9,6 @@ configdir=@sysconfdir@/NetworkManager
 Name: NetworkManager
 Description: System for maintaining active network connection
 Requires:
+Requires.private: glib-2.0
 Version: @VERSION@
 Cflags: -I${includedir}/NetworkManager
-- 
1.9.3

